.github/workflows/release.yaml

```diff
@@ -121,6 +122,7 @@ jobs:
       name: "prod"
     container: "hashicorp/terraform:0.14.10"
     needs: [build, prod-plan]
+    if: needs.test.outputs.changed == true
```

Comment:
Should this be `needs.build.outputs.changed` ?

Reply:
yes, good catch